spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -179,6 +181,10 @@ main() {
     # teardown will happen automatically on exit
 }
 
+MOVED_TO_TEST_CONTAINERS=(
```

Comment:
I'd like to move one module at a time to testcontainers, so this just tracks what project is already moved. When all of them will be moved, I will drop this, obviously.